lib/telemetry_metrics/reporter/last_value.ex

```diff
@@ -0,0 +1,23 @@
+defmodule Membrane.TelemetryMetrics.Reporter.LastValue do
+  @moduledoc false
+
+  alias Membrane.TelemetryMetrics.Utils
+
+  @spec attach(Telemetry.Metrics.LastValue.t(), :ets.tid() | atom()) :: [reference()]
+  def attach(metric, ets) do
+    config = %{ets: ets, measurement: metric.measurement}
+    Utils.attach_metric_handler(metric.event_name, &__MODULE__.handle_event/4, config)
+  end
+
+  @spec handle_event([atom(), ...], map(), map(), term()) :: :ok
+  def handle_event(_event_name, measurements, metadata, config) do
+    %{ets: ets, measurement: measurement} = config
+
+    with %{membrane_telemetry_label: label} <- metadata,
+         %{^measurement => value} <- measurements do
+      :ets.insert(ets, {label, value})
+    end
```

Comment:
shouldn't these be normal matches?

Reply:
Do you mean matches in function definition? I have added them in function body, because even if somebody will emit event with measurements without expected key, I don't want from this function to fail (also, writing this specific pattern match as above in function definition seemed to be l
ittle difficult)